The problem was the build:
```
npm ERR! material-ui-pagination@1.0.2 build: `./node_modules/webpack/bin/webpack.js --progress --colors  --config webpack.config.js`
```
It does work if I install webpack globally and change the build script to `webpack ...` instead of `node_modules/webpack/bin/webpack.js ...` (even though webpack is in the `node_modules` folder.

--- a/src/index.jsx
+++ b/src/index.jsx
@@ -142,13 +142,13 @@ class Pagination extends React.Component {
 Pagination.propTypes = {
 
   // eslint-disable-next-line react/no-unused-prop-types
-  total: propTypes.number,
+  total: PropTypes.number,
 
   // eslint-disable-next-line react/no-unused-prop-types
-  current: propTypes.number,
+  current: PropTypes.number,
 
   // eslint-disable-next-line react/no-unused-prop-types
-  display: propTypes.number,
-  onChange: propTypes.func,
+  display: PropTypes.number,
+  onChange: PropTypes.func,
 };
 export default Pagination;

